src/celeritas/ext/detail/ImportProcessConverter.cc

```diff
@@ -148,68 +150,11 @@ ImportModelClass to_import_model(G4VEmModel const& model)
         static celeritas::TypeDemangler<G4VEmModel> demangle_model;
         CELER_LOG(warning) << "Encountered unknown model '" << name
                            << "' (RTTI: " << demangle_model(model) << ")";
-        return ImportModelClass::unknown;
+        return ImportModelClass::none;
     }
     return iter->second;
 }
 
-//---------------------------------------------------------------------------//
-/*!
- * Safely switch from \c G4PhysicsVectorType to \c ImportPhysicsVectorType .
- * [See G4PhysicsVectorType.hh]
- *
- * Geant4 v11 has a different set of G4PhysicsVectorType enums.
- */
-ImportPhysicsVectorType
-to_import_physics_vector_type(G4PhysicsVectorType g4_vector_type)
-{
-    switch (g4_vector_type)
-    {
-#if CELERITAS_G4_V10
-        case T_G4PhysicsVector:
-            return ImportPhysicsVectorType::unknown;
-#endif
-        case T_G4PhysicsLinearVector:
-            return ImportPhysicsVectorType::linear;
-        case T_G4PhysicsLogVector:
-#if CELERITAS_G4_V10
-        case T_G4PhysicsLnVector:
-#endif
-            return ImportPhysicsVectorType::log;
-        case T_G4PhysicsFreeVector:
-#if CELERITAS_G4_V10
-        case T_G4PhysicsOrderedFreeVector:
-        case T_G4LPhysicsFreeVector:
-#endif
```

Comment:
I see why this is organized this way, but it might be easier to read (at the cost of more lines) if we had
```cpp
#if CELERITAS_G4_V10
    [G4 v10 cases]
#else
    [G4 v11 cases]
#endif
```

Reply:
Even better, how about we drop support for removed V10 enums?

Reply:
Note that I only *moved* the lines in this PR: this implementation was already in place. How about we plan a separate PR to remove support for Geant4@10-only features?

Reply:
That sounds good to me.